src/instrumentsscene/qml/MuseScore/InstrumentsScene/ChooseInstrumentsPage.qml

```diff
@@ -59,6 +59,10 @@ Rectangle {
         familyView.focusOnFirst()
     }
 
+    function focusOnFirstSearchField() {
```

Comment:
Please rename as focusOnFirst

Reply:
Is this correct? Before that function there is another one named focusOnFirst

Reply:
Ok
So you don't need a second function
Just replace the body of the function to instrumentsView.focusOnFirst()
(And remove redundant function, like familyView.focusOnFirst())

Reply:
Ok all right